lib/cc/cli/command.rb

```diff
@@ -43,6 +43,10 @@ def require_codeclimate_yml
         end
       end
 
+      def set_local_owner(file= ".codeclimate.yml")
```

Comment:
I don't think there's a lot of value in the default here. It duplicates a constant and may run into issues because we're never calling `FileSystem#path_for`. I would drop it and either always call this method with `filesystem.path_for(CODECLIMATE_YML)` or implement `FileSystem#stat` and have *it* call `#path_for` internally, maybe?